fiware-region-sanity-tests/commons/constants.py

```diff
@@ -26,6 +26,7 @@
 
 # CONFIGURATION PROPERTIES
 PROPERTIES_FILE = "resources/settings.json"
+PROPERTIES_LOGGER = "TestCase"
```

Comment:
I see no need to change this: "TestCase" means logs come from class "unittest.TestCase" or any of its subclasses. This way the logger differentiates from other novaclient.* or neutronclient.* loggers.